tests/data/validation_problems.po

```diff
@@ -49,3 +49,17 @@ msgstr[1] "2. Estas {num} objectos"
 # msgid_plural "3. There are {num} things"
 # msgstr[0] "3. Estas num objectos"
 # msgstr[1] "3. Estas {num} objectos"
+
+# These should not raise a validation error, because LONG_DATE_FORMAT and
+# friends are special.
+msgid "LONG_DATE_FORMAT"
```

Comment:
This is a low-tech way to identify these names.  I guess I could just make a list instead?